src/instruments/instruments.controller.ts

```diff
@@ -142,12 +139,10 @@ export class InstrumentsController {
         { _id: id },
         updateInstrumentDto,
       );
+
       return instrument;
-    } catch (e) {
-      throw new HttpException(
-        "Instrument with the same unique name already exists",
-        HttpStatus.BAD_REQUEST,
-      );
+    } catch (error) {
+      throw new InternalServerErrorException(error);
```

Comment:
Does this exception expose the internal error to the user?
If so, I would like to change it so we log the internal error and we provide the same error as before.